### PR TITLE
feat: [CCM-7119]: Recommendation List Page Enhancements

### PR DESCRIPTION
**Pull request by rajarshee-harness**
##### Jira Links:

https://harness.atlassian.net/browse/CCM-7119

##### Screenshots:

Screenshot 2022-03-29 at 10 33 47 AM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by rajarshee-harness** [comment]
trigger rebase

File: src/modules/75-ce/pages/recommendationList/RecommendationList.tsx [hunk 1]
```diff
@@ -87,6 +83,14 @@ const RecommendationsList: React.FC<RecommendationListProps> = ({
     }
   }, [])
 
+  const resourceTypeMap: Record<string, string> = useMemo(
+    () => ({
+      [ResourceType.Workload]: getString('ce.overview.workload'),
+      [ResourceType.NodePool]: getString('ce.overview.nodepool')
+    }),
+    []
```
Comment (diptiman-adak) [inline src/modules/75-ce/pages/recommendationList/RecommendationList.tsx:91]: Codacy has a fix for the issue: [React Hook useMemo has a missing dependency: 'getString'. Either include it or remove the dependency array.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232309)
```suggestion
    [getString]
```

**Comment by rajarshee-harness** [comment]
retrigger jest

File: src/modules/75-ce/pages/recommendationList/RecommendationList.tsx (hunk 1 above)
Comment (diptiman-adak) [inline src/modules/75-ce/pages/recommendationList/RecommendationList.tsx:91]: Codacy has a fix for the issue: [React Hook useMemo has a missing dependency: 'getString'. Either include it or remove the dependency array.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232309)
```suggestion
    [getString]
```

File: src/modules/75-ce/components/RecommendationSavingsCard/RecommendationSavingsCard.tsx [hunk 2]
```diff
@@ -5,35 +5,49 @@
...
-        >
-          {amount}
-        </Text>
-        {subTitle ? <Text font="small">{subTitle}</Text> : null}
+        <Container>
+          <Text
+            inline
+            font={{ variation: FontVariation.H3 }}
+            color={iconName ? Color.GREEN_700 : Color.GREY_800}
+            icon={iconName ? iconName : undefined}
```
Comment (rishik30) [inline src/modules/75-ce/components/RecommendationSavingsCard/RecommendationSavingsCard.tsx:35]: you can just use `iconName`. No need to have a conditional operator for `undefined`

**Review by rishik30 — APPROVED** [review]

File: src/modules/75-ce/pages/recommendationList/RecommendationList.tsx (hunk 1 above)
Comment (diptiman-adak) [inline src/modules/75-ce/pages/recommendationList/RecommendationList.tsx:91]: Codacy has a fix for the issue: [React Hook useMemo has a missing dependency: 'getString'. Either include it or remove the dependency array.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232309)
```suggestion
    [getString]
```

**Review by vkbansal — APPROVED** [review]

**Pull request merged**